startup/40-hdf5.py

```diff
@@ -381,10 +343,8 @@ def process_packing_queue():
             if db[uid].stop['exit_status'] != 'success': # the scan actually finished
                 print(f"scan {uid} was not successful.")
                 return 
-        #print(uid)
-        #print(path)
-        PilatusCBFHandler.froot = data_file_path[frn]
-        threading.Thread(target=pack_and_move, args=(data_type,uid,path,move_first,)).start() 
+
+        threading.Thread(target=pack_and_process, args=(data_type,uid,path,move_first,)).start() 
```

Comment:
Check my comment about it above.
```suggestion
        threading.Thread(target=pack_and_process, args=(data_type, uid, path,)).start() 
```

Reply:
fixed in commit https://github.com/NSLS-II-LIX/profile_collection/pull/13/commits/9cd622d019226132fe7ada84779fa970e27a3ed2

Reply:
I believe this was fixed as part of https://github.com/NSLS-II-LIX/profile_collection/pull/13/commits/9cd622d019226132fe7ada84779fa970e27a3ed2